bgpd/bgpd.h

```diff
@@ -792,6 +810,19 @@ struct bgp_filter {
 		char *name;
 		struct route_map *map;
 	} usmap;
+
+	/* Advertise-map */
+	struct {
+		char *aname;
+		struct route_map *amap;
+
+		bool condition;
+
+		char *cname;
+		struct route_map *cmap;
+
+		enum update_type advertise;
```

Comment:
nit: advertise -> update_type adds more readability into code itself

Reply:
changed advertise to update_type.